### PR TITLE
Make --token optional (through .gitlabrc file)

### PR DESCRIPTION
**Pull request by brendo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,53 @@ Run the following command from the directory that contains the properties file,
...
+Set NPM_INSTALL_TOKEN = <value> for gitlab-org/gitlab-ce.
+Completed setting variables on Gitlab CI.
+```
+
+Essentially, if your project is a git repository, and you have `.gitlabrc` file,
+no options are required and this tool can be invoked simply as:
+
+```sh
+$ setAllVars
+No URL specified, using git remote `origin`.
```
Comment (quangkhoa) [inline README.md:86]: I wonder whether the app should display the same message for URL, e.g. `not token provided, using token found under .gitlabrc`.

File: README.md [hunk 2]
```diff
@@ -37,18 +37,53 @@ Run the following command from the directory that contains the properties file,
...
+Completed setting variables on Gitlab CI.
 ```
 
 > Omitting `--url` will derive the URL from the remote named `origin`.
+
+This project supports `.gitlabrc` files using [rc](https://www.npmjs.com/package/rc).
+If `--token` is not specified, this project can use a `.gitlabrc`, e.g:
+
+```ini
+token: this-is-my-gitlab-token
```
Comment (quangkhoa) [inline README.md]: should be `=` instead of `:`

Comment (brendo) [reply README.md]: Good catch!

File: README.md (hunk 1 above)
Comment (brendo) [reply README.md:86]: Will do

**Review by quangkhoa — APPROVED** [review]

**Pull request merged**